spec/models/accounting/qb/purchase_extractor_spec.rb

```diff
@@ -39,7 +40,24 @@
       expect(txn.account).to eq txn_acct
       expect(txn.amount).to equal_money(12345.67)
       expect(txn.sync_token).to eq "abc"
-      expect(txn.qb_object_subtype).to eq "Check"
+      expect(txn.disbursement_type).to eq "check"
+      expect { txn.save! }.not_to raise_error
+    end
+  end
+
+  context "extracting a non-check" do
+    let(:payment_type) { "Cash" }
+    it "updates correctly in Madeline" do
+      Accounting::QB::PurchaseExtractor.new(txn).extract!
+      expect(txn.loan_transaction_type_value).to eq "disbursement"
+      expect(txn.managed).to be false
+      expect(txn.line_items.size).to eq 2
+      expect(txn.line_items[1].account).to eq txn.account
+      expect(txn.line_items[1].credit?).to be true
+      expect(txn.account).to eq txn_acct
+      expect(txn.amount).to equal_money(12345.67)
+      expect(txn.sync_token).to eq "abc"
+      expect(txn.disbursement_type).to eq "other"
```

Comment:
Thanks for adding spec coverage! Great use of context blocks with the payment_type let var. Textbook!